js/__internal/grids/grid_core/keyboard_navigation/m_keyboard_navigation_utils.ts

```diff
@@ -7,16 +7,21 @@ import {
   MASTER_DETAIL_ROW_CLASS, VIRTUAL_ROW_CLASS,
 } from './const';
 
+const DATAGRID_GROUP_FOOTER_CLASS = 'dx-datagrid-group-footer';
+
 export function isGroupRow($row) {
   return $row && $row.hasClass(GROUP_ROW_CLASS);
 }
+export function isGroupFooterRow($row) {
+  return $row && $row.hasClass(DATAGRID_GROUP_FOOTER_CLASS);
+}
 
 export function isDetailRow($row) {
   return $row && $row.hasClass(MASTER_DETAIL_ROW_CLASS);
 }
 
 export function isDataRow($row) {
-  return $row && !isGroupRow($row) && !isDetailRow($row);
+  return $row && !isGroupRow($row) && !isGroupFooterRow($row) && !isDetailRow($row);
```

Comment:
Question: does data row contain its own class like `dx-data-row`? If yes, we need to rewrite this function to use it

Reply:
You are right, I fixed it.